PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+**What does this PR do?**
+
+xxx
+
+**Description of Task to be completed?**
+
+xxx
+
+**How should this be manually tested?**
+
+xxx
+
+**Any background context you want to provide?**
+
+xxx
+
+**What are the relevant pivotal tracker stories?**
+
+xxx
+
+**Screenshots (if appropriate)**
+
+xxx
+
+**Questions:**
+
+xxx
```

Comment:
leave an extra line

Reply:
There's an extra line, Github removes it when displaying the file. If there wasn't you would see a red arrow under need the file.

Reply:
Got it. Rebase from develop